core-bundle/assets/backend.js

```diff
@@ -50,3 +50,6 @@ document.documentElement.addEventListener('turbo:frame-missing', (e) => {
     e.preventDefault();
     e.detail.visit(e.detail.response);
 });
+
+/* Styles */
```

Comment:
```suggestion
// Styles
```

Reply:
I removed the comment in 230eabb7adbea0e91804553128b11ddd6ab03d02.